### PR TITLE
Feature/np 4

### PR DESCRIPTION
**Pull request by felipegaete**
np-4
registered users list and user detail component

### CODE REVIEW

**Review by lcaloguerea — APPROVED** [review]

File: apps/dashboard/src/app/dashboard/components/user/user.component.scss [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    }
+  }
+  &__about {
+    padding: 15px 0px;
+    @include sectionBorderBottom;
+    &__title {
+      @include text(20px, $gray, bold);
+    }
+    &__description {
+      margin-top: 8px !important;
```
Comment (ruedagato) [inline apps/dashboard/src/app/dashboard/components/user/user.component.scss:55]: is necessary the important?

Comment (ruedagato) [inline apps/dashboard/src/app/dashboard/components/user/user.component.scss:55]: is necessary the important?

File: apps/dashboard/src/app/dashboard/components/user/user.component.scss [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    left bottom,
+    color-stop(0%, $startColor),
+    color-stop(24%, $startColor),
+    color-stop(87%, $endColor),
+    color-stop(100%, $endColor)
+  );
+  background: -webkit-linear-gradient(top, $startColor 0%, $startColor 24%, $endColor 87%, $endColor 100%);
+  background: -o-linear-gradient(top, $startColor 0%, $startColor 24%, $endColor 87%, $endColor 100%);
+  background: -ms-linear-gradient(top, $startColor 0%, $startColor 24%, $endColor 87%, $endColor 100%);
+  background: linear-gradient(to bottom, $startColor 0%, $startColor 24%, $endColor 87%, $endColor 100%);
```
Comment (ruedagato) [inline apps/dashboard/src/app/dashboard/components/user/user.component.scss:99]: you only need this line the other is auto generate

**Review by ruedagato — COMMENTED** [review]
fix this warning, please
Captura de Pantalla 2021-02-17 a la(s) 5 56 24 p  m

**Pull request merged**